code/datums/ai/sanity/sanityloss_behaviors.dm

```diff
@@ -175,8 +175,8 @@
 	if(DT_PROB(5, delta_time) && living_pawn.Adjacent(target) && isturf(target.loc))
 		living_pawn.visible_message("<span class='danger'>[living_pawn] smashes panel on \the [target]!</span>")
 		playsound(living_pawn.loc, 'sound/effects/hit_on_shattered_glass.ogg', 75, TRUE, -1)
-		if(!target.finish_work(living_pawn))
-			finish_action(controller, TRUE)
+		target.datum_reference.qliphoth_change(-9)
```

Comment:
This is not how you should do it.
First of all - it should only reduce qliphoth by 1.
Secondly - there must be a check for 0 qliphoth later, before finishing action.